automat/_test/test_visualize.py

```diff
@@ -1,16 +1,34 @@
-
-from __future__ import unicode_literals
+import functools
 
 import os
 import subprocess
 from unittest import TestCase, skipIf
 
+from characteristic import attributes
+
 from .._methodical import MethodicalMachine
 
+from .._visualize import elementMaker, tableMaker
+
+
+def isGraphvizModuleInstalled():
+    """
+    Is the graphviz Python module installed?
+    """
+    try:
+        import graphviz
+    except ImportError:
+        return False
+    else:
+        del graphviz
```

Comment:
On the contrary, I feel like `__import__('graphviz')` is actually *more* clear; it unambiguously says "I'm importing this for the side-effect, not to bind the name".  So I'd go that direction.